### DISCUSSION TITLE
No puedo ver las transmisión en teams linux

### DISCUSSION THREAD

**Discussion by RobertoSalgadoC** [Launch-X-Latam/OnBoarding#12] [Q&A]
Me aparece el siguiente error 
No está habilitada la opción para unirte a un evento en vivo con el codificador externo.

Esto me aparece desde teams para linux
Uso ubuntu budgie 22.04

**Comment by devonent** [comment]
Y si accedes desde el navegador también te sale el mismo mensaje de error?

**Reply by RobertoSalgadoC** [reply]
En el navegador no, pero carga lento o no se si la transmisión de ayer se estaba trabando mucho

**Reply by devonent** [reply]
Si, la transmisión de ayer se traba mucho, después se arregla un poco, no es por tu internet o algo parecido, así quedó grabado, no te preocupes 👍.

**Reply by Gl096** [reply]
¿saben como ingresar a las sesiones en vivo?
